tests/Meilisearch.Tests/DocumentTests.cs

```diff
@@ -89,6 +126,24 @@ public async Task BasicDocumentsUpdate()
             docs.ElementAt(1).Genre.Should().BeNull();
         }
 
+        [Fact]
+        public async Task DocumentsUpdateWithPrimaryKey()
+        {
+            var indexUid = "DocumentsUpdateWithPrimaryKeyTest";
+            var index = this.client.Index(indexUid);
+            index.PrimaryKey.Should().BeNull();
+
+            // Add the documents
+            var document = await index.UpdateDocuments(new[] { new { movieId = "1", Name = "Batman" } }, "movieId");
```

Comment:
Use a primary key that does not contain id

Reply:
Done